doc/usage.rst

```diff
@@ -293,71 +134,55 @@ Let's find out more about the datasets:
 
 Now we can restrict the tasks to all tasks with the desired resampling strategy:
 
-# TODO add something about the different resampling strategies implemented!
-
 .. code:: python
-    >>> filtered_tasks = tasks.query('estimation_procedure == "10-fold Crossvalidation"')
-    >>> filtered_tasks = list(filtered_tasks.index)
-    >>> print(filtered_tasks)                               # doctest: +SKIP
-    [1, 2, 3, 4, 5, 6, 7, 8, 9, ... 10105, 10106, 10107, 10109, 10111, 13907, 13918]
 
-Resampling strategies can be found on the `OpenML Website <http://www.openml.org/search?type=measure&q=estimation%20procedure>`_
-or programatically as described in `Finding out evaluation strategies and target metrics`_.
-
-Finally, we can check whether there is a task for each dataset that we want to
-use in our study. If this is not the case, tasks can be created on the
-`OpenML website <openml.org/tasks/create>`_.
+    >>> filtered_tasks = tasks.query('estimation_procedure == "10-fold Crossvalidation"')
+    >>> print(list(filtered_tasks.index))                               # doctest: +SKIP
+    [2, 3, 4, 5, 6, 7, 8, 9, ..., 146606, 146607, 146690]
+    >>> print(len(filtered_tasks))                                      # doctest: +SKIP
+    1697
 
-The rest of this subsection deals with accessing a list of tasks by tags and
-without any restriction.
+Resampling strategies can be found on the `OpenML Website <http://www.openml.org/search?type=measure&q=estimation%20procedure>`_.
 
-A list of tasks, filtered tags, can be retrieved via:
+We can further filter the list of tasks to only contain datasets with more than
+500 samples, but less than 1000 samples:
 
 .. code:: python
 
-    >>> tasks = openml.tasks.list_tasks(tag='study_1')
+    >>> filtered_tasks = filtered_tasks.query('NumberOfInstances > 500 and NumberOfInstances < 1000')
```

Comment:
Or just move this up, this seems more natural then the CV type to me? Or motivate the CV type?